docs/stellar-disbursement-platform/getting-started.mdx

```diff
@@ -193,15 +193,15 @@ Now that you've been able to make a disbursement, lets go back to our docker com
 
 ### Email and SMS Messages
 
-The Stellar Disbursement Platform sends SMS messages and emails to receivers and users, respectively. For SMS messages, the Stellar Disbursemnet Platform supports Twilio or AWS SNS. For emails, it supports AWS SES.
+The Stellar Disbursement Platform sends SMS messages and emails to receivers and users, respectively. For SMS messages, the Stellar Disbursement Platform supports Twilio, AWS SNS, and a dry run mode that just logs the messages. For emails, it supports AWS SES and dry run.
 
-You'll need to chose if you want to use Twilio or AWS SNS for SMS messages. Once you create your AWS and potentially your Twilio account, you can configure the following values with your credentials.
+These services can be selected through the `SMS_SENDER_TYPE` and `EMAIL_SENDER_TYPE` configurations. When selecting Twilio or AWS services, you'll need to fill their service-specific configuration as well. Here's an example of what the setup might look like if you're using both Twilio and AWS.
```

Comment:
very minor. add colon at the end to 
`Here's an example of what the setup might look like if you're using both Twilio and AWS:`